### PR TITLE
add support for python-3.11 with mixed casing in end if/for/while keywords

### PR DESCRIPTION
**Pull request by john-tornblom**
How about this?

### CODE REVIEW

**Comment by leviathan747** [comment]
Change '3.10' to '3.x' in these locations:

https://github.com/xtuml/pyxtuml/blob/1faa1d801a2d844836da56a46c913cff3016e994/.github/workflows/build-release-artifacts.yaml#L21
https://github.com/xtuml/pyxtuml/blob/1faa1d801a2d844836da56a46c913cff3016e994/.github/workflows/publish-pypi-release.yaml#L18
https://github.com/xtuml/pyxtuml/blob/1faa1d801a2d844836da56a46c913cff3016e994/.github/workflows/run-tests.yaml#L18

File: .github/workflows/run-tests.yaml [hunk 1]
```diff
@@ -15,7 +15,7 @@ jobs:
     strategy:
       matrix:
         os:               [ubuntu-latest, windows-latest]
-        python-version:   ['2.7', '3.10', 'pypy2.7', 'pypy3.8']
+        python-version:   ['2.7', '3.x', 'pypy2.7', 'pypy3.x']
```
Comment (leviathan747) [inline .github/workflows/run-tests.yaml]: For some reason, the GitHub action for setting up python does not resolve ".x" for pypy versions.

**Comment by john-tornblom** [comment]
Seems to work now (pypy3.x did not work with github actions).

**Comment by john-tornblom** [comment]
Squashed into a single commit...

**Pull request merged**